R/user_helpers.R

```diff
@@ -73,8 +81,15 @@ use_ad_template <- function(adam_name = "adsl",
 #'
 #' @examples
 #' list_all_templates()
-list_all_templates <- function() {
-  list.files(system.file("templates", package = "admiral")) %>%
+list_all_templates <- function(package = "admiral") {
```

Comment:
If no templates are found, the output is
```
Existing templates:
• 
```
I would prefer something like "No templates available in package xxx".

Reply:
Agree.

Reply:
Now looking like this:

```r
list_all_templates("admiral")
```
```
Existing ADaM templates in package 'admiral':
• ADAE
• ADCM
• ADEG
• ADEX
• ADLB
• ADPP
• ADSL
• ADVS
```
and
```r
list_all_templates("dplyr")
```
```
No ADaM templates available in package 'dplyr'
```